@xen-orchestra/web-core/lib/tables/column-sets/vm-backup-archive-columns.ts

```diff
@@ -0,0 +1,15 @@
+import { defineColumns } from '@core/packages/table/define-columns.ts'
+import { useDateColumn } from '@core/tables/column-definitions/date-column'
+import { useLinkColumn } from '@core/tables/column-definitions/link-column'
+import { useNumberColumn } from '@core/tables/column-definitions/number-column'
+import { useI18n } from 'vue-i18n'
+
+export const useBackuparchiveColumns = defineColumns(() => {
+  const { t } = useI18n()
+
+  return {
+    date: useDateColumn({ headerLabel: () => t('network') }),
```

Comment:
```suggestion
    date: useDateColumn({ headerLabel: () => t('date'), dateStyle: 'short', timeStyle: 'medium' }),
```

It's not exactly the same date format as on Figma, but it's very close (maybe the date format should be changed for everything, I don't know).